Transporter.IntegrationTests/Tests/Couchbase-Couchbase/CouchbaseToCouchbaseIntegrationTests.cs

```diff
@@ -0,0 +1,146 @@
+using System;
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Threading;
+using System.Threading.Tasks;
+using AutoFixture;
+using Dapper;
+using FluentAssertions;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Moq;
+using NUnit.Framework;
+using Transporter.Core.Utils;
+using Transporter.CouchbaseAdapter.Data.Interfaces;
+using Transporter.IntegrationTests.Base;
+using Transporter.IntegrationTests.Constants;
+using Transporter.IntegrationTests.Helpers.Couchbase.Interfaces;
+using Transporter.IntegrationTests.Objects;
+using TransporterService;
+
+namespace Transporter.IntegrationTests.Tests.Couchbase_Couchbase
+{
+    public class CouchbaseToCouchbaseIntegrationTests : IntegrationTestBase
+    {
+        private Fixture _fixture;
+        private Mock<IDateTimeProvider> _mockDateTimeProvider;
+        [SetUp]
+        public void SetUp()
+        {
+            _fixture = new Fixture();
+            _mockDateTimeProvider = new Mock<IDateTimeProvider>();
+            _mockDateTimeProvider.Setup(m => m.Now).Returns(DateTime.UtcNow.AddMinutes(-5));
+        }
+
+        [Test]
+        public async Task Transporter_ShouldTransportData()
+        {
+            //Arrange
+            var id = CreateFixture<Guid>().ToString();
+            var age = CreateFixture<int>();
+            var collection =  await GetRequiredService<IBucketProvider>()
+                .GetBucket(GetCouchbaseConnectionData(), JobConstants.CouchbaseToCouchbaseJob.SourceBucketName);
+            var couchbaseCollection =    await collection.DefaultCollectionAsync();
+            await GetRequiredService<ICouchbaseProviderHelper>()
+                .CreatePrimaryIndexAsync(GetCouchbaseConnectionData(), $"`{JobConstants.CouchbaseToCouchbaseJob.SourceBucketName}`");
+            await GetRequiredService<ICouchbaseProviderHelper>()
+                .CreateBucketAsync(GetCouchbaseConnectionData(), JobConstants.CouchbaseToCouchbaseJob.InterimBucketName);
+            await GetRequiredService<ICouchbaseProviderHelper>()
+                .CreateBucketAsync(GetCouchbaseConnectionData(), JobConstants.CouchbaseToCouchbaseJob.TargetBucketName);
+            await GetRequiredService<ICouchbaseProviderHelper>()
+                .CreateIndexAsync(GetCouchbaseConnectionData(), $"`{JobConstants.CouchbaseToCouchbaseJob.InterimBucketName}`", JobConstants.CouchbaseToCouchbaseJob.InterimBucketIndexName, new List<string>
+                {
+                    "dataSourceName",
+                    "lmd"
+                });
+            
+            var user = _fixture.Build<UserClass>().With(x=> x.Age, age).Create();
```

Comment:
how about we don't specify age and use "user.age" in verify section?

Reply:
verification changed as user.Age